lib/saluki-health/src/lib.rs

```diff
@@ -64,38 +70,80 @@ impl Health {
     }
 }
 
-#[derive(Eq, PartialEq)]
+#[derive(Clone, Copy, Eq, PartialEq)]
```

Comment:
Is this utilized anywhere?

Reply:
In `ComponentState::mark_live`, `ComponentState::mark_not_live`, and `ComponentState::mark_dead`, we pass `self.health` to `Telemetry::update_liveness`, which required making `HealthState` either `Copy` or passing it by reference... I just chose to make it `Copy` since it's not more efficient to pass it by reference.

Mostly just stylistic.